### ISSUE TITLE
WinJS.UI.Pivot: Why do I need to style .win-pivot .win-pivot-surface to see items?

### ISSUE THREAD

**Issue by pke** [winjs/winjs#1035]
Without setting

```
.win-pivot .win-pivot-surface {
    height: 75vh;
}
```

None of the `PivotItems` display any content. Once I set this style rule everything seems to work. That is directly from the VS Phone templates and I don't see it in the try site.

**Comment by TheWrathOfConst** [comment]
This is an issue with the VS template. In general, developers shouldn't have to style the height of `.win-pivot .win-pivot-surface`. The pivot should size itself to fit the space it's given.

The problem in the VS template's case is that the Pivot can't size itself to fit the space it's given, because its ancestors are styled such that they expect their children to expand and fit the space they need. Since the Pivot doesn't do that by default, it looks like the template added in that rule to force it to.

To fix this, we had to put heights on these two elements:
```html
<div class="hubpage fragment" style="height: 100%;">
   <section role="main" style="height: 100%;" aria-label="Main content">
```
Once that was done, the height: 75vh rule could be removed.

Thanks for reporting this. We've forwarded this along to the VS team
